CMakeLists.txt

```diff
@@ -253,6 +251,13 @@ add_compile_options(/MP) # Build with Multiple Processes (number of processes eq
 add_compile_options(/GS) # Buffer Security Check
 add_compile_options(/Zm200) # Specify Precompiled Header Memory Allocation Limit of 150MB
 add_compile_options(/wd4960 /wd4961 /wd4603 /wd4627 /wd4838 /wd4456 /wd4457 /wd4458 /wd4459 /wd4091 /we4640)
+add_compile_options(/O1) # minimize space
+
+if (CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL Relwithdebinfo)
+  add_compile_options(/GL) # enable WholeProgramOptimization
```

Comment:
Ok. :)

If you add a comment on main PR view (https://github.com/dotnet/coreclr/pull/1111) with message: `@dotnet-bot test this please`, it will re-trigger the CI build. You can do as many times as you want. You can also add `/O2` amend and re-push, and then remove and re-push to confirm whether or not `/O2` it is the source of issue here.